packages/SwingSet/test/upgrade/test-upgrade.js

```diff
@@ -565,15 +546,16 @@ test('non-durable exports are abandoned by upgrade of liveslots vat', async t =>
   };
   /** @type {Record<string, { presence: unknown, kref: string }>} */
   const counters = {};
-  const runIncrement = presence =>
-    run('messageVatObject', [{ presence, methodName: 'increment' }]);
+  const runIncrement = presence => messageToObject(presence, 'increment');
   for (const [name, methodName] of Object.entries(counterGetters)) {
-    const counter = await run('messageVat', [{ name: 'exporter', methodName }]);
+    const counter = await messageToObject(exporterRoot, methodName);
     const val = await runIncrement(counter);
     t.is(val, 1, `initial increment of ${name}`);
-    const counterRef = await run('awaitVatObject', [
-      { presence: counter, rawOutput: true },
-    ]);
+    const counterRef = await messageToVat(
+      'bootstrap',
+      'awaitVatObject',
+      counter,
+    );
     const kref = krefOf(counterRef);
     counters[name] = { presence: counter, kref };
```

Comment:
Is this `awaitVatObject` still necessary?
```suggestion
    counters[name] = { presence: counter, kref: krefOf(counter) };
```
which points out that we shouldn't even need the `{ presence, kref }` structure.
```js
  for (const [name, methodName] of Object.entries(counterGetters)) {
    const counter = await messageToObject(exporterRoot, methodName);
    …
    counters[name] = counter;
  }
  …
  const ephCounterRC1 = findRowByKref(rc1, krefOf(counters.ephCounter));
  …
```

Reply:
Perhaps if someone could explain to me what `awaitVatObject` actually _does?_  This was actually somewhat mysterious to me, so I just propagated it through with only superficial changes.  But if we could eliminate it that would also take  care of the issue of having it in the non-test version of the bootstrap vat.